examples/align-gl/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(RealsenseExamplesAlignGl )
+
+# Save the command line compile commands in the build output
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
```

Comment:
What do you mean by `save` ?

Reply:
I don't remember we use this line in other examples

Reply:
I didn't realize this. It got included when I took reference `CMakelists.txt` from [rs-gl](https://github.com/IntelRealSense/librealsense/blob/development/examples/gl/CMakeLists.txt#L9) example

Reply:
OK, not a big deal but looks redundent 
`If enabled, generates a compile_commands.json file...`